subsys/fs/shell.c

```diff
@@ -46,6 +54,204 @@ static int cmd_mkdir(int argc, char *argv[])
 	return 0;
 }
 
+static int cmd_rf(int argc, char *argv[])
```

Comment:
why not rm? this is more consistent with the other command you are using.

Reply:
right, I'll change this